models/cow_protocol/ethereum/cow_protocol_ethereum_trades_slippage.sql

```diff
@@ -8,26 +8,16 @@
 with 
 raw_data as (
     select
+    order_uid
     block_number,
     block_time,
     case when (flags % 2) = 0 then 'SELL' else 'BUY' end  as order_type,
-    limit_buy_amount,
     CASE when (flags % 2) = 0 THEN (limit_buy_amount / (1.0 - (slippage_bips / 10000.0))) else (limit_buy_amount) end as buy_quote,
     atoms_bought,
-    atoms_bought / pow(10, A.decimals) as units_bought,
-    buy_price,
-    A.decimals as buy_decimals,
-    limit_sell_amount,
     CASE when (flags % 2) = 1 THEN (limit_sell_amount / (1.0 + (slippage_bips / 10000.0))) else (limit_sell_amount) end as sell_quote,
     atoms_sold,
-    atoms_sold / pow(10, B.decimals) as units_sold,
-    sell_price,
-    B.decimals as sell_decimals,
-    usd_value,
-    slippage_bips,
-    (1.0 - (slippage_bips / 10000.0)) as buy_tolerance,
-    (1.0 + (slippage_bips / 10000.0)) as sell_tolerance,
-    order_uid
```

Comment:
Seems like a lot is being removed... where are they going?

Reply:
They weren't used before and were forgotten in here.